atd-etl/cris_import/cris_import.py

```diff
@@ -727,13 +777,12 @@ def group_csvs_into_logical_groups(extracted_archives, dry_run):
     files = os.listdir(str(extracted_archives))
     logical_groups = []
     for file in files:
-        if file.endswith(".xml"):
+        if file.endswith(".xml") or file == "crashReports":
```

Comment:
Extracts can have a folder in them of PDFs now, and we don't want to treat the folder with the code for CSVs in this block, so this skips over that folder.